QuickConnectPlugin/ArgumentsFormatters/RemoteDesktopArgumentsFormatter.cs

```diff
@@ -31,6 +41,23 @@ public class RemoteDesktopArgumentsFormatter : IArgumentsFormatter {
                 }
             }
 
+            sb.AppendFormat(" /u:\"{0}\"", hostPwEntry.GetUsername());
+            sb.AppendFormat(" /p:\"{0}\"", hostPwEntry.GetPassword());
+
+            if (xUseRdpPlus) {
+                RdpPlusOptions options;
+                if (RdpPlusOptionsParser.TryParse(hostPwEntry.AdditionalOptions, out options)) {
+                    if (!string.IsNullOrEmpty(options.RDGateway)) {
+                        var extraOptions = "gatewayhostname:s:" + options.RDGateway + ", gatewayusagemethod:i:2, gatewaycredentialssource:i:4, gatewayprofileusagemethod:i:1, gatewaybrokeringtype:i:0";
```

Comment:
Those are passed to the /o argument of RDP+, which basically means "pass this on to ms rdp".
What I did was save the rdp file from mstsc without gateway and with gateway, and added those options here.